sysutils/ondir/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             ondir
+version          0.2.3
+revision         1
```

Comment:
The revision should start at 0.

Reply:
ciao @mf2k! thanks for your revision.
i fixed the revision number.